microprofile.jdt/com.redhat.microprofile.jdt.core/src/main/java/com/redhat/microprofile/jdt/core/java/IJavaDiagnosticsParticipant.java

```diff
@@ -0,0 +1,55 @@
+/*******************************************************************************
+* Copyright (c) 2020 Red Hat Inc. and others.
+* All rights reserved. This program and the accompanying materials
+* which accompanies this distribution, and is available at
+* http://www.eclipse.org/legal/epl-v20.html
+*
+* Contributors:
+*     Red Hat Inc. - initial API and implementation
+*******************************************************************************/
+package com.redhat.microprofile.jdt.core.java;
+
+import org.eclipse.core.runtime.CoreException;
+import org.eclipse.core.runtime.IProgressMonitor;
+import org.eclipse.jdt.core.JavaModelException;
+
+/**
+ * Java diagnostics participants API.
+ * 
+ * @author Angelo ZERR
+ *
+ */
+public interface IJavaDiagnosticsParticipant {
+
+	/**
+	 * Returns true if the collection of diagnostics must be done for the given

```

Comment:
... true if diagnostics must be collected for the given ...

Reply:
fixed.